platform/android/java/src/org/godotengine/godot/GodotView.java

```diff
@@ -114,6 +114,13 @@ public boolean onTouchEvent(MotionEvent event) {
 		return activity.gotTouchEvent(event);
 	};
 
+	@Override
+	public boolean onHoverEvent(MotionEvent event) {
+		super.onHoverEvent(event);
+		return activity.gotTouchEvent(event);
+	};
+
+
```

Comment:
`clang-format` complains about this line, our style rules disallow multiple consecutive empty lines.